lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -14,6 +14,8 @@ module Rails
         #   refute_equal(a, b)
         #   assert_nil a
         #   refute_empty(b)
+        #   assert_true(a)
+        #   assert_false b
```

Comment:
Does it make sense to also detect plain `assert`/`refute` from Minitest?

Reply:
This is a technical note. Minitest's `assert` is compatible with `be_truthy`, and `refute` is compatible with `be_falsey`. This means that the behavior of Test::Unit's `assert_true` / `assert_false` and Minitest's `assert` / `refute` are different.

- Test::Unit ... https://github.com/test-unit/test-unit/blob/3.6.1/lib/test/unit/assertions.rb#L1379
- Minitest ... https://github.com/minitest/minitest/blob/v5.21.2/lib/minitest/assertions.rb#L180

Reply:
Thanks for bringing this up!